web-client/src/presenter/actions/StartCaseInternal/serveCaseToIrsAction.ts

```diff
@@ -16,16 +16,20 @@ export const serveCaseToIrsAction = async ({
   props,
 }: ActionProps) => {
   const docketNumber = props.docketNumber || get(state.caseDetail.docketNumber);
+  const clientConnectionId = get(state.clientConnectionId);
 
-  const pdfUrl = await applicationContext
-    .getUseCases()
-    .serveCaseToIrsInteractor(applicationContext, {
-      docketNumber,
+  try {
```

Comment:
Now that this is async and the real work is happening behind the scenes with the lambda I don't think that we need the try catch around this interactor call because all of our async calls respond back that they work initially.

Reply:
🤔 Hmm, I'm a little confused. How else would I report to the client when an error occurs in the async lambda? 

The async lambdas report immediately to the client, and then they rely on a websocket message to relay successes, failures, or progress updates as necessary.

Reply:
In an async lambda the api gateway immediately responds successfully before the lambda's interactor has even started. So the try/catch here can only really catch if the user's computer does not have internet access or if our API is down which can happen but is just unlikely.

Like you said it is in the websocket messages that all of the success, errors, or progress is communicated. So to answer your question the websockets is where you would report to the client that an error has occurred in the async lambda.

Reply:
Yes, I'm sorry. I get it now. For some odd reason I thought this was referring to the try ... catch block in the Interactor, not the Action. 

Yes, I agree this would really only catch whether or not the user's computer has internet access. And should that happen, I think there's probably a better strategy to employ to handle failed network requests.